<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="REHNA_NAZEER" failures="2" tests="4" name="Test" time="50.464" errors="0" timestamp="2025-04-01T11:57:57 IST">
  <testcase classname="testScript.LoginTest" name="verifyUserLoginWithValidCredentials" time="2.556"/>
  <testcase classname="testScript.LoginTest" name="verifyUserLoginWithValidUsernameAndInvalidPassword" time="1.066"/>
  <testcase classname="testScript.LoginTest" name="verifyUserLoginWithInvalidUsernameAndValidPassword" time="1.015">
    <failure type="java.lang.AssertionError" message="User is able to login with invalid password expected [7rmart Supermarket] but found [7rmart supermarket]">
      <![CDATA[java.lang.AssertionError: User is able to login with invalid password expected [7rmart Supermarket] but found [7rmart supermarket]
at testScript.LoginTest.verifyUserLoginWithInvalidUsernameAndValidPassword(LoginTest.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserLoginWithInvalidUsernameAndValidPassword -->
  <testcase classname="testScript.LoginTest" name="verifyUserLoginWithInvalidCredentials" time="1.007">
    <failure type="java.lang.AssertionError" message="User is able to login with invalid credentials expected [7rmart Supermarket] but found [7rmart supermarket]">
      <![CDATA[java.lang.AssertionError: User is able to login with invalid credentials expected [7rmart Supermarket] but found [7rmart supermarket]
at testScript.LoginTest.verifyUserLoginWithInvalidCredentials(LoginTest.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserLoginWithInvalidCredentials -->
</testsuite> <!-- Test -->
